### PR TITLE
Add module kat hist

### LINKED ISSUE #887: new module: KAT/HIST
I think it would be good to have a module for [KAT](https://bioconda.github.io/recipes/kat/README.html)

- [x] This module does not exist yet with the [`nf-core modules list`](https://github.com/nf-core/tools#list-modules) command
- [x] There is no [open pull request](https://github.com/nf-core/modules/pulls) for this module
- [x] There is no [open issue](https://github.com/nf-core/modules/issues) for this module
- [x] If I'm planning to work on this module, I added myself to the `Assignees` to facilitate tracking who is working on the module

### PR DESCRIPTION
**Pull request by mahesh-panchal**
Add's the module KAT hist to plot k-mer histograms on sequence data. 

## PR checklist

Closes #887 

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [x] Remove all TODO statements.
- [x] Emit the `versions.yml` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [x] Follow the input/output options guidelines.
- [x] Add a resource `label`
- [x] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

**Comment by mahesh-panchal** [comment]
Current issue:
Pytest throwing a stack trace after doing:

```
nf-core modules create-test-yml kat/hist
```
Remove the `contains` line with the TODO in the yaml since it's a binary file
and then run
```
PROFILE=docker pytest --tag kat/hist --symlink --kwd --git-aware
```
which throws an error:
```
$ PROFILE=docker pytest --tag kat/hist --symlink --kwd --git-aware
================================================================================================================================================ test session starts =================================================================================================================================================
platform linux -- Python 3.9.7, pytest-7.1.1, pluggy-1.0.0
rootdir: /workspace/modules, configfile: pytest.ini
plugins: workflow-1.6.0
collecting ... 
collecting 0 items                                                                                                                                                                                                                                                                               collecting 797 items                                                                                                                                                                                                                                                                             collecting 797 items                                                                                                                                                                                                                                                                             collecting 797 items                                                                                                                                                                                                                                                                             collecting 816 items                                                                                                                                                                                                                                                                             collecting 816 items                                                                                                                                                                                                                                                                             collecting 816 items                                                                                                                                                                                                                                                                             collected 816 items                                                                                                                                                                                                                                                                                                  

kat hist test_kat_hist_single_end:
        command:   nextflow run tests/modules/kat/hist -entry test_kat_hist_single_end -c tests/config/nextflow.config
        directory: /tmp/pytest_workflow_p0sqabe8/kat_hist_test_kat_hist_single_end
        stdout:    /tmp/pytest_workflow_p0sqabe8/kat_hist_test_kat_hist_single_end/log.out
        stderr:    /tmp/pytest_workflow_p0sqabe8/kat_hist_test_kat_hist_single_end/log.err
'kat hist test_kat_hist_single_end' done.

kat hist test_kat_hist_paired_end:
        command:   nextflow run tests/modules/kat/hist -entry test_kat_hist_paired_end -c tests/config/nextflow.config
        directory: /tmp/pytest_workflow_p0sqabe8/kat_hist_test_kat_hist_paired_end
        stdout:    /tmp/pytest_workflow_p0sqabe8/kat_hist_test_kat_hist_paired_end/log.out
        stderr:    /tmp/pytest_workflow_p0sqabe8/kat_hist_test_kat_hist_paired_end/log.err
'kat hist test_kat_hist_paired_end' done.

tests/test_versions_yml.py sssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss [ 33%]
ssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssFFssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss [ 70%]
ssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss                                                                                   [ 97%]
tests/modules/kat/hist/test.yml FFFFFFFFFFFFFFFFFFF                                                                                                                                                                                                                                                            [100%] Keeping temporary directories and logs. Use '--kwd' or '--keep-workflow-wd' to disable this behaviour.


====================================================================================================================================================== FAILURES ======================================================================================================================================================
__________________________________________________________________________________________________________________________ test_ensure_valid_version_yml[kat hist test_kat_hist_single_end] __________________________________________________________________________________________________________________________

workflow_dir = PosixPath('/tmp/pytest_workflow_p0sqabe8/kat_hist_test_kat_hist_single_end')

    @pytest.mark.workflow(*_get_workflow_names())
    def test_ensure_valid_version_yml(workflow_dir):
        workflow_dir = Path(workflow_dir)
        software_name = workflow_dir.name.split("_")[0].lower()
        try:
            versions_yml_file = workflow_dir / f"output/{software_name}/versions.yml"
>           versions_yml = versions_yml_file.read_text()

tests/test_versions_yml.py:31: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = PosixPath('/tmp/pytest_workflow_p0sqabe8/kat_hist_test_kat_hist_single_end/output/kat/versions.yml'), encoding = None, errors = None

    def read_text(self, encoding=None, errors=None):
        """
        Open the file in text mode, read it, and close the file.
        """
>       with self.open(mode='r', encoding=encoding, errors=errors) as f:

/opt/conda/lib/python3.9/pathlib.py:1266: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = PosixPath('/tmp/pytest_workflow_p0sqabe8/kat_hist_test_kat_hist_single_end/output/kat/versions.yml'), mode = 'r', buffering = -1, encoding = None, errors = None, newline = None

    def open(self, mode='r', buffering=-1, encoding=None,
             errors=None, newline=None):
        """
        Open the file pointed by this path and return a file object, as
        the built-in open() function does.
        """
>       return io.open(self, mode, buffering, encoding, errors, newline,
                       opener=self._opener)

/opt/conda/lib/python3.9/pathlib.py:1252: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = PosixPath('/tmp/pytest_workflow_p0sqabe8/kat_hist_test_kat_hist_single_end/output/kat/versions.yml'), name = '/tmp/pytest_workflow_p0sqabe8/kat_hist_test_kat_hist_single_end/output/kat/versions.yml', flags = 524288, mode = 438

    def _opener(self, name, flags, mode=0o666):
        # A stub for the opener argument to built-in open()
>       return self._accessor.open(self, flags, mode)
E       FileNotFoundError: [Errno 2] No such file or directory: '/tmp/pytest_workflow_p0sqabe8/kat_hist_test_kat_hist_single_end/output/kat/versions.yml'

/opt/conda/lib/python3.9/pathlib.py:1120: FileNotFoundError

During handling of the above exception, another exception occurred:

workflow_dir = PosixPath('/tmp/pytest_workflow_p0sqabe8/kat_hist_test_kat_hist_single_end')

    @pytest.mark.workflow(*_get_workflow_names())
    def test_ensure_valid_version_yml(workflow_dir):
        workflow_dir = Path(workflow_dir)
        software_name = workflow_dir.name.split("_")[0].lower()
        try:
            versions_yml_file = workflow_dir / f"output/{software_name}/versions.yml"
            versions_yml = versions_yml_file.read_text()
        except FileNotFoundError:
>           raise AssertionError(
                dedent(
                    f"""\
                    `versions.yml` not found in the output directory.
                    Expected path: `{versions_yml_file}`
    
                    This can have multiple reasons:
                    * The test-workflow failed before a `versions.yml` could be generated.
                    * The workflow name in `test.yml` does not start with the tool name.
                    """
                )
            )
E           AssertionError: `versions.yml` not found in the output directory.
E           Expected path: `/tmp/pytest_workflow_p0sqabe8/kat_hist_test_kat_hist_single_end/output/kat/versions.yml`
E           
E           This can have multiple reasons:
E           * The test-workflow failed before a `versions.yml` could be generated.
E           * The workflow name in `test.yml` does not start with the tool name.

tests/test_versions_yml.py:33: AssertionError
__________________________________________________________________________________________________________________________ test_ensure_valid_version_yml[kat hist test_kat_hist_paired_end] __________________________________________________________________________________________________________________________

workflow_dir = PosixPath('/tmp/pytest_workflow_p0sqabe8/kat_hist_test_kat_hist_paired_end')

    @pytest.mark.workflow(*_get_workflow_names())
    def test_ensure_valid_version_yml(workflow_dir):
        workflow_dir = Path(workflow_dir)
        software_name = workflow_dir.name.split("_")[0].lower()
        try:
            versions_yml_file = workflow_dir / f"output/{software_name}/versions.yml"
>           versions_yml = versions_yml_file.read_text()

tests/test_versions_yml.py:31: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = PosixPath('/tmp/pytest_workflow_p0sqabe8/kat_hist_test_kat_hist_paired_end/output/kat/versions.yml'), encoding = None, errors = None

    def read_text(self, encoding=None, errors=None):
        """
        Open the file in text mode, read it, and close the file.
        """
>       with self.open(mode='r', encoding=encoding, errors=errors) as f:

/opt/conda/lib/python3.9/pathlib.py:1266: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = PosixPath('/tmp/pytest_workflow_p0sqabe8/kat_hist_test_kat_hist_paired_end/output/kat/versions.yml'), mode = 'r', buffering = -1, encoding = None, errors = None, newline = None

    def open(self, mode='r', buffering=-1, encoding=None,
             errors=None, newline=None):
        """
        Open the file pointed by this path and return a file object, as
        the built-in open() function does.
        """
>       return io.open(self, mode, buffering, encoding, errors, newline,
                       opener=self._opener)

/opt/conda/lib/python3.9/pathlib.py:1252: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = PosixPath('/tmp/pytest_workflow_p0sqabe8/kat_hist_test_kat_hist_paired_end/output/kat/versions.yml'), name = '/tmp/pytest_workflow_p0sqabe8/kat_hist_test_kat_hist_paired_end/output/kat/versions.yml', flags = 524288, mode = 438

    def _opener(self, name, flags, mode=0o666):
        # A stub for the opener argument to built-in open()
>       return self._accessor.open(self, flags, mode)
E       FileNotFoundError: [Errno 2] No such file or directory: '/tmp/pytest_workflow_p0sqabe8/kat_hist_test_kat_hist_paired_end/output/kat/versions.yml'

/opt/conda/lib/python3.9/pathlib.py:1120: FileNotFoundError

During handling of the above exception, another exception occurred:

workflow_dir = PosixPath('/tmp/pytest_workflow_p0sqabe8/kat_hist_test_kat_hist_paired_end')

    @pytest.mark.workflow(*_get_workflow_names())
    def test_ensure_valid_version_yml(workflow_dir):
        workflow_dir = Path(workflow_dir)
        software_name = workflow_dir.name.split("_")[0].lower()
        try:
            versions_yml_file = workflow_dir / f"output/{software_name}/versions.yml"
            versions_yml = versions_yml_file.read_text()
        except FileNotFoundError:
>           raise AssertionError(
                dedent(
                    f"""\
                    `versions.yml` not found in the output directory.
                    Expected path: `{versions_yml_file}`
    
                    This can have multiple reasons:
                    * The test-workflow failed before a `versions.yml` could be generated.
                    * The workflow name in `test.yml` does not start with the tool name.
                    """
                )
            )
E           AssertionError: `versions.yml` not found in the output directory.
E           Expected path: `/tmp/pytest_workflow_p0sqabe8/kat_hist_test_kat_hist_paired_end/output/kat/versions.yml`
E           
E           This can have multiple reasons:
E           * The test-workflow failed before a `versions.yml` could be generated.
E           * The workflow name in `test.yml` does not start with the tool name.

tests/test_versions_yml.py:33: AssertionError
____________________________________________________________________________________________________________________________________________________ test session ____________________________________________________________________________________________________________________________________________________
'/tmp/pytest_workflow_p0sqabe8/kat_hist_test_kat_hist_single_end/output/kat/test.hist' does not exist while it should
____________________________________________________________________________________________________________________________________________________ test session ____________________________________________________________________________________________________________________________________________________
Observed md5sum 'None' not equal to expected md5sum '1aceb823b6774f14f5cca15954138efd' for file '/tmp/pytest_workflow_p0sqabe8/kat_hist_test_kat_hist_single_end/output/kat/test.hist'
____________________________________________________________________________________________________________________________________________________ test session ____________________________________________________________________________________________________________________________________________________
'/tmp/pytest_workflow_p0sqabe8/kat_hist_test_kat_hist_single_end/output/kat/test.hist-hash.jf27' does not exist while it should
____________________________________________________________________________________________________________________________________________________ test session ____________________________________________________________________________________________________________________________________________________
'/tmp/pytest_workflow_p0sqabe8/kat_hist_test_kat_hist_single_end/output/kat/test.hist.dist_analysis.json' does not exist while it should
____________________________________________________________________________________________________________________________________________________ test session ____________________________________________________________________________________________________________________________________________________
Observed md5sum 'None' not equal to expected md5sum 'ec4317d510f752855411d13c0f24dea9' for file '/tmp/pytest_workflow_p0sqabe8/kat_hist_test_kat_hist_single_end/output/kat/test.hist.dist_analysis.json'
____________________________________________________________________________________________________________________________________________________ test session ____________________________________________________________________________________________________________________________________________________
'/tmp/pytest_workflow_p0sqabe8/kat_hist_test_kat_hist_single_end/output/kat/test.hist.png' does not exist while it should
____________________________________________________________________________________________________________________________________________________ test session ____________________________________________________________________________________________________________________________________________________
Observed md5sum 'None' not equal to expected md5sum '30daaf4d2dd4b1574e387df973c39d03' for file '/tmp/pytest_workflow_p0sqabe8/kat_hist_test_kat_hist_single_end/output/kat/test.hist.png'
____________________________________________________________________________________________________________________________________________________ test session ____________________________________________________________________________________________________________________________________________________
'/tmp/pytest_workflow_p0sqabe8/kat_hist_test_kat_hist_single_end/output/kat/versions.yml' does not exist while it should
____________________________________________________________________________________________________________________________________________________ test session ____________________________________________________________________________________________________________________________________________________
Observed md5sum 'None' not equal to expected md5sum '296f28c007bc55f8f5490702cf3f81d4' for file '/tmp/pytest_workflow_p0sqabe8/kat_hist_test_kat_hist_single_end/output/kat/versions.yml'
____________________________________________________________________________________________________________________________________________________ test session ____________________________________________________________________________________________________________________________________________________
'kat hist test_kat_hist_single_end' exited with exit code '1' instead of '0'.
____________________________________________________________________________________________________________________________________________________ test session ____________________________________________________________________________________________________________________________________________________
'/tmp/pytest_workflow_p0sqabe8/kat_hist_test_kat_hist_paired_end/output/kat/test.hist' does not exist while it should
____________________________________________________________________________________________________________________________________________________ test session ____________________________________________________________________________________________________________________________________________________
Observed md5sum 'None' not equal to expected md5sum 'c4e8e01996bd8f676e02690220e1def7' for file '/tmp/pytest_workflow_p0sqabe8/kat_hist_test_kat_hist_paired_end/output/kat/test.hist'
____________________________________________________________________________________________________________________________________________________ test session ____________________________________________________________________________________________________________________________________________________
'/tmp/pytest_workflow_p0sqabe8/kat_hist_test_kat_hist_paired_end/output/kat/test.hist.dist_analysis.json' does not exist while it should
____________________________________________________________________________________________________________________________________________________ test session ____________________________________________________________________________________________________________________________________________________
Observed md5sum 'None' not equal to expected md5sum '954a3d1c3fc56f80f0929315ed299824' for file '/tmp/pytest_workflow_p0sqabe8/kat_hist_test_kat_hist_paired_end/output/kat/test.hist.dist_analysis.json'
____________________________________________________________________________________________________________________________________________________ test session ____________________________________________________________________________________________________________________________________________________
'/tmp/pytest_workflow_p0sqabe8/kat_hist_test_kat_hist_paired_end/output/kat/test.hist.png' does not exist while it should
____________________________________________________________________________________________________________________________________________________ test session ____________________________________________________________________________________________________________________________________________________
Observed md5sum 'None' not equal to expected md5sum '8a21ce0965342234f5982f947f4f10b0' for file '/tmp/pytest_workflow_p0sqabe8/kat_hist_test_kat_hist_paired_end/output/kat/test.hist.png'
____________________________________________________________________________________________________________________________________________________ test session ____________________________________________________________________________________________________________________________________________________
'/tmp/pytest_workflow_p0sqabe8/kat_hist_test_kat_hist_paired_end/output/kat/versions.yml' does not exist while it should
____________________________________________________________________________________________________________________________________________________ test session ____________________________________________________________________________________________________________________________________________________
Observed md5sum 'None' not equal to expected md5sum '3a07329af7ce78e76b021996cd08eacb' for file '/tmp/pytest_workflow_p0sqabe8/kat_hist_test_kat_hist_paired_end/output/kat/versions.yml'
____________________________________________________________________________________________________________________________________________________ test session ____________________________________________________________________________________________________________________________________________________
'kat hist test_kat_hist_paired_end' exited with exit code '1' instead of '0'.
============================================================================================================================================== short test summary info ===============================================================================================================================================
FAILED tests/test_versions_yml.py::test_ensure_valid_version_yml[kat hist test_kat_hist_single_end] - AssertionError: `versions.yml` not found in the output directory.
FAILED tests/test_versions_yml.py::test_ensure_valid_version_yml[kat hist test_kat_hist_paired_end] - AssertionError: `versions.yml` not found in the output directory.
FAILED tests/modules/kat/hist/test.yml::kat hist test_kat_hist_single_end::output/kat/test.hist::should exist
FAILED tests/modules/kat/hist/test.yml::kat hist test_kat_hist_single_end::output/kat/test.hist::md5sum
FAILED tests/modules/kat/hist/test.yml::kat hist test_kat_hist_single_end::output/kat/test.hist-hash.jf27::should exist
FAILED tests/modules/kat/hist/test.yml::kat hist test_kat_hist_single_end::output/kat/test.hist.dist_analysis.json::should exist
FAILED tests/modules/kat/hist/test.yml::kat hist test_kat_hist_single_end::output/kat/test.hist.dist_analysis.json::md5sum
FAILED tests/modules/kat/hist/test.yml::kat hist test_kat_hist_single_end::output/kat/test.hist.png::should exist
FAILED tests/modules/kat/hist/test.yml::kat hist test_kat_hist_single_end::output/kat/test.hist.png::md5sum
FAILED tests/modules/kat/hist/test.yml::kat hist test_kat_hist_single_end::output/kat/versions.yml::should exist
FAILED tests/modules/kat/hist/test.yml::kat hist test_kat_hist_single_end::output/kat/versions.yml::md5sum
FAILED tests/modules/kat/hist/test.yml::kat hist test_kat_hist_single_end::exit code should be 0
FAILED tests/modules/kat/hist/test.yml::kat hist test_kat_hist_paired_end::output/kat/test.hist::should exist
FAILED tests/modules/kat/hist/test.yml::kat hist test_kat_hist_paired_end::output/kat/test.hist::md5sum
FAILED tests/modules/kat/hist/test.yml::kat hist test_kat_hist_paired_end::output/kat/test.hist.dist_analysis.json::should exist
FAILED tests/modules/kat/hist/test.yml::kat hist test_kat_hist_paired_end::output/kat/test.hist.dist_analysis.json::md5sum
FAILED tests/modules/kat/hist/test.yml::kat hist test_kat_hist_paired_end::output/kat/test.hist.png::should exist
FAILED tests/modules/kat/hist/test.yml::kat hist test_kat_hist_paired_end::output/kat/test.hist.png::md5sum
FAILED tests/modules/kat/hist/test.yml::kat hist test_kat_hist_paired_end::output/kat/versions.yml::should exist
FAILED tests/modules/kat/hist/test.yml::kat hist test_kat_hist_paired_end::output/kat/versions.yml::md5sum
FAILED tests/modules/kat/hist/test.yml::kat hist test_kat_hist_paired_end::exit code should be 0
========================================================================================================================================== 21 failed, 795 skipped in 8.07s ===========================================================================================================================================
g
```

File: tests/modules/kat/hist/test.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  files:
+    - path: output/kat/test.hist
+      md5sum: 1aceb823b6774f14f5cca15954138efd
+    - path: output/kat/test.hist-hash.jf27
+    - path: output/kat/test.hist.dist_analysis.json
+      md5sum: ec4317d510f752855411d13c0f24dea9
+    - path: output/kat/test.hist.png
+      md5sum: 30daaf4d2dd4b1574e387df973c39d03
+    - path: output/kat/versions.yml
+      md5sum: 296f28c007bc55f8f5490702cf3f81d4
```
Comment (FriederikeHanssen) [inline tests/modules/kat/hist/test.yml:15]: I think we have agreed to avoid md5sums for versions. Julia already added this to nf-core/tools dev.

Comment (mahesh-panchal) [reply tests/modules/kat/hist/test.yml:15]: 👍🏽 I'll run the dev version then.

Comment (FriederikeHanssen) [reply tests/modules/kat/hist/test.yml:15]: could also just delete that line

Comment (mahesh-panchal) [reply tests/modules/kat/hist/test.yml:15]: true
```suggestion
```

File: tests/modules/kat/hist/test.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    - kat
+  files:
+    - path: output/kat/test.hist
+      md5sum: c4e8e01996bd8f676e02690220e1def7
+    - path: output/kat/test.hist.dist_analysis.json
+      md5sum: 954a3d1c3fc56f80f0929315ed299824
+    - path: output/kat/test.hist.png
+      md5sum: 8a21ce0965342234f5982f947f4f10b0
+    - path: output/kat/versions.yml
+      md5sum: 3a07329af7ce78e76b021996cd08eacb
```
Comment (mahesh-panchal) [inline tests/modules/kat/hist/test.yml:30]:
```suggestion
```

File: tests/modules/kat/hist/test.yml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        - "nb_peaks"
+        - "global_minima"
+        - "global_maxima"
+        - "mean_freq"
+        - "est_genome_size"
+        - "est_het_rate"
+    - path: output/kat/test.hist.png
+      md5sum: 49861ef1a265e0edde3550b39c64a274
+    - path: output/kat/versions.yml
+      md5sum: 296f28c007bc55f8f5490702cf3f81d4
```
Comment (sateeshblue) [inline tests/modules/kat/hist/test.yml:22]:
```suggestion

```

File: tests/modules/kat/hist/test.yml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+        - "nb_peaks"
+        - "global_minima"
+        - "global_maxima"
+        - "mean_freq"
+        - "est_genome_size"
+        - "est_het_rate"
+    - path: output/kat/test.hist.png
+      md5sum: e20774d0d2b979cb6ead7b7fb5ad36d9
+    - path: output/kat/versions.yml
+      md5sum: 3a07329af7ce78e76b021996cd08eacb
```
Comment (sateeshblue) [inline tests/modules/kat/hist/test.yml:44]:
```suggestion

```

**Review by sateeshblue — APPROVED** [review]
rest lgtm!

**Pull request merged**